File: spec/code_samples/C_libcurl/lists/get.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
curl_easy_setopt(hnd, CURLOPT_URL, "https://analyzemail.com/api/v1/lists?username=SOME_STRING_VALUE");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "content-type: application/json");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

CURLcode ret = curl_easy_perform(hnd);

File: spec/code_samples/Javascript_jquery/lists@cleaned/get.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://analyzemail.com/api/v1/lists/cleaned?username=SOME_STRING_VALUE",
  "method": "GET",
  "headers": {
    "content-type": "application/json"
  }
}

$.ajax(settings).done(function (response) {
  console.log(response);
});